### PR TITLE
Code refinements, better error handling

### PR DESCRIPTION
**Pull request by PuruVJ**
## Description
Some code prettification, better error handling in some places

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by proful** [comment]
@Mira-Alf @tolgaerdonmez are you guys fine with this changes?

File: twindle-cli/pdf/create-pdf.js [hunk 1]
```diff
@@ -3,19 +3,19 @@ const fs = require("fs");
 
 // Creates a pdf document from htmlContent and saves it to outputPath
 async function createPdf(outputPath, htmlContent) {
-	// launchs a puppeteer browser instance and opens a new page
-	const browser = await puppeteer.launch();
-	const page = await browser.newPage();
+  // launchs a puppeteer browser instance and opens a new page
+  const browser = await puppeteer.launch({ args: ["--no-sandbox"] });
```
Comment (PuruVJ) [inline twindle-cli/pdf/create-pdf.js:7]: This was required for my WSL environment and it is also required when this script will be turned over to a Serverless environment

Comment (tolgaerdonmez) [reply twindle-cli/pdf/create-pdf.js:7]: Yeah very True, also it is needed to run the code in linux servers

**Review by tolgaerdonmez — APPROVED** [review]
Most of them are actually indentation but, some are needed

File: twindle-cli/pdf/create-pdf.js (hunk 1 above)
Comment (Mira-Alf) [reply twindle-cli/pdf/create-pdf.js:7]: When I applied these changes locally the PDF file was not being generated, though no errors appeared.

File: twindle-cli/twitter/utils/tweet-lookup-request.js [hunk 2]
```diff
@@ -1,54 +1,86 @@
...
-        await processResponse(response);
-    } catch(err) {
-        console.log(err);
-    }
+  try {
+    // console.log(getUrl(tweet_id));
+    const response = await fetch(getUrl(tweet_id), getRequestOptions());
+    await processResponse(response);
+  } catch (err) {
+    console.err(err);
```
Comment (Mira-Alf) [inline twindle-cli/twitter/utils/tweet-lookup-request.js]: Change this to console.error. err is not a function it complains.

File: twindle-cli/pdf/create-pdf.js (hunk 1 above)
Comment (PuruVJ) [reply twindle-cli/pdf/create-pdf.js:7]: no-sandbox doesn't seem to be working on Windows.

**Comment by proful** [comment]
@Mira-Alf is it working? If not breaking any thing I will merge it

**Comment by PuruVJ** [comment]
More additions. The code works now. Issues were with putting await at the wrong place

**Pull request merged**